lisa/operating_system.py

```diff
@@ -298,9 +298,10 @@ def install_packages(
         packages: Union[str, Tool, Type[Tool], List[Union[str, Tool, Type[Tool]]]],
         signed: bool = True,
         timeout: int = 600,
+        backports: bool = False,
```

Comment:
It looks the backports applies to Debian only. Please rename it to `extra` or `additional_arguments` and support it in all os.